doc.ddoc

```diff
@@ -1,3 +1,4 @@
+ROOT_DIR =
```

Comment:
The point is `ROOT_SLASH` is more descriptive. Good makefile style is to NOT append slashes to directories in macros because on the use side it can be easily added and it's difficult to remove.